client/src/game/states.ts

```diff
@@ -9,7 +9,10 @@ export const states = {
     // when created or re-assigned
     gameSocket: undefined as (Socket | undefined),
     lobby: undefined as (types.LobbyId | undefined),
+    players: ref<Map<types.LobbyId, string>>(new Map()),
     id: undefined as (types.ClientId | undefined),
+    isAdmin: ref<boolean>(false),
```

Comment:
its called owner in the backend, not really necessary to rename but it will help mantain solidity across the project.